### PR TITLE
Unit tests for safely surrendered babies

### PR DESCRIPTION
**Pull request by saleemis-cwds**
## Description

Unit tests for safely surrendered babies

## Jira Issue link

https://osi-cwds.atlassian.net/browse/HOT-1982

## Tests
- [x] I have included unit tests 
- [ ] I have included functional tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the google code style of this project.
- [x] I have ran all tests for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check SonarQube after the build completes, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/rest/services/SpecialProjectReferralService.java [hunk 1]
```diff
@@ -52,35 +55,43 @@ public SpecialProjectReferralService() {
...
   public void processSafelySurrenderedBabies(String childClientId, String referralId,
+      LocalDate referralReceivedDate, LocalTime referralReceivedTime,
       gov.ca.cwds.rest.api.domain.SafelySurrenderedBabies ssb) {
 
     LocalDateTime now = LocalDateTime.now();
-    String userId = PrincipalUtils.getStaffPersonId();
-    String countyCode = PrincipalUtils.getPrincipal().getCountyCode();
-    String countyCwsCode = PrincipalUtils.getPrincipal().getCountyCwsCode();
 
+    PerryUserIdentity perruUser = RequestExecutionContext.instance().getUserIdentity();
```
Comment (mariamcws) [inline src/main/java/gov/ca/cwds/rest/services/SpecialProjectReferralService.java]: Small typo, I think you meant perryUser not perruUser :-)

File: src/test/java/gov/ca/cwds/rest/api/domain/ScreeningToReferralTest.java [hunk 2]
```diff
@@ -219,35 +213,39 @@ public void shouldValidateWithValid() throws Exception {
...
-    ScreeningToReferral toValidate = new ScreeningToReferralResourceBuilder().setParticipants(null).createScreeningToReferral();
+    ScreeningToReferral toValidate =
+        new ScreeningToReferralResourceBuilder().setParticipants(null).createScreeningToReferral();
     validator = Validation.buildDefaultValidatorFactory().getValidator();
     messageBuilder.addDomainValidationError(validator.validate(toValidate));
     Boolean theErrorDetected = false;
     List<ErrorMessage> validationErrors = messageBuilder.getMessages();
     for (ErrorMessage message : validationErrors) {
-//      System.out.println(message.getMessage());
+      // System.out.println(message.getMessage());
```
Comment (mariamcws) [inline src/test/java/gov/ca/cwds/rest/api/domain/ScreeningToReferralTest.java]: there are a few of these commented out System.out.println , is this on purpose.

File: src/main/java/gov/ca/cwds/rest/services/SpecialProjectReferralService.java (hunk 1 above)
Comment (saleemis-cwds) [reply src/main/java/gov/ca/cwds/rest/services/SpecialProjectReferralService.java]: Thanks for catching, will fix.

File: src/test/java/gov/ca/cwds/rest/api/domain/ScreeningToReferralTest.java (hunk 2 above)
Comment (saleemis-cwds) [reply src/test/java/gov/ca/cwds/rest/api/domain/ScreeningToReferralTest.java]: These statements should be changed to use logger. They have been there already, I'll try to cleanup.

**Pull request merged**